### PR TITLE
Make it dreamy to run robusta-runner locally

### PR DESCRIPTION
**Pull request by aantn**
@robusta-dev ❤️ @metalbear-co

### CODE REVIEW

File: run_runner_locally.sh [hunk 1]
```diff
@@ -6,7 +6,10 @@ set -e
 
 # Settings you might want to change
 export PYTHON_BINARY=python3.9
-export PORT=5000
+export PORT=6000  # if you change this, update port_mapping in mirrord.json too
```
Comment (arikalon1) [inline run_runner_locally.sh:9]: Was there a reason for changing the port?

File: run_runner_locally.sh [hunk 2]
```diff
@@ -6,7 +6,10 @@ set -e
...
+export LOG_LEVEL=INFO
+#export TRACE_INCOMING_REQUESTS=false
```
Comment (arikalon1) [inline run_runner_locally.sh:11]: can we just delete?

File: run_runner_locally.sh [hunk 3]
```diff
@@ -58,15 +61,13 @@ if [ ! -f "$PWD/deployment/playbooks/active_playbooks.yaml" ]; then
 fi
 
 echo "Installing builtin playbooks"
-poetry run python3 -m pip install -e ./deployment/playbooks/defaults
+#poetry run python3 -m pip install -e ./deployment/playbooks/defaults
```
Comment (arikalon1) [inline run_runner_locally.sh:64]: Why did you remove the local playbooks installation? Isn't that required for dev?

File: run_runner_locally.sh [hunk 4]
```diff
@@ -58,15 +61,13 @@ if [ ! -f "$PWD/deployment/playbooks/active_playbooks.yaml" ]; then
...
 export PLAYBOOKS_ROOT=./deployment/playbooks
 export REPO_LOCAL_BASE_DIR=./deployment/git_playbooks
 export INSTALLATION_NAMESPACE=default
-export TRACE_INCOMING_REQUESTS=true
-export LOG_LEVEL=DEBUG
-
-poetry run python3 -m robusta.runner.main
 
+mirrord exec -f mirrord.json -- poetry run python3 -m robusta.runner.main
+#poetry run python3 -m robusta.runner.main
```
Comment (arikalon1) [inline run_runner_locally.sh:73]: can we just delete?

**Review by arikalon1 — APPROVED** [review]
Looks good
Left some minor comments.
Feel free to fix and merge

File: run_runner_locally.sh (hunk 1 above)
Comment (aantn) [reply run_runner_locally.sh:9]: That port is already taken on my laptop, I think it happened to someone else too. If you don't mind, I'd like to jjust change the default.

File: run_runner_locally.sh (hunk 2 above)
Comment (aantn) [reply run_runner_locally.sh]: I want to leave it here so people see and know they can change it easily.

File: run_runner_locally.sh (hunk 3 above)
Comment (aantn) [reply run_runner_locally.sh]: Whoops, fixed.

File: run_runner_locally.sh (hunk 4 above)
Comment (aantn) [reply run_runner_locally.sh:72]: Yes, done.
